lib/webhooks/src/request-handlers/post.js

```diff
@@ -84,6 +84,10 @@ class PostHandler extends BaseHandler {
 
         const msg = newEmptyMessage();
         msg.metadata.externalExecId = this.getRequestId();
+        msg.metadata.source = {
+            id: 'webhooks',
```

Comment:
Why wouldn't we but `this.getRequestId()` here? Is there a reason that's a separate metadata field from this ID?